include/deal.II/differentiation/ad/ad_number_types.h

```diff
@@ -31,10 +31,21 @@ namespace Differentiation
      * is to be used for computations. If a type that is selected for use
      * is not available in the library, a run-time error will be thrown.
      *
-     * @author Jean-Paul Pelteret, 2017
+     * @author Jean-Paul Pelteret, 2017, 2019
      */
     enum class NumberTypes
     {
+      /**
+       * A dummy type for floating point numbers (0-differentiable).
```

Comment:
I think "0-differentiable" is not a term that's commonly understood. Can you elaborate?